src/frontend/context/ProjectInviteContext.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+
+export interface ProjectInvite {
+  project: {
+    id: string;
+    name: string;
+  };
+  role: string;
+}
+
+const ProjectInviteContext = React.createContext<{
+  invite?: ProjectInvite;
+  removeInvite: (inviteId: string) => void;
+}>({ removeInvite: () => null });
```

Comment:
are you talking about the initialization or the actual `removeInvite` function? 

if the former, you need to initialize the context with a valid value. decided that a no-op instead of it just not existing would be a better way to go.

if the latter, it's so that we can process the invite after some action occurs (such as a declining the invite)

Reply:
I honestly don't remember this comment, but it looks fine now that I'm looking at it again.